src/backend/launcher.ts

```diff
@@ -66,16 +66,6 @@ async function prepareLaunch(
     const epicNonReachable = !isOnline() || (await isEpicServiceOffline())
     // If the game is configured to use offline mode or Epic isn't reachable, but the game can't run offline, we can't launch
     if (gameSettings.offlineMode || epicNonReachable) {
-      showErrorBoxModalAuto(
-        i18next.t(
-          'box.error.no-offline-mode.title',
-          'Offline mode not supported.'
-        ),
-        i18next.t(
-          'box.error.no-offline-mode.message',
-          'Launch aborted! The game requires a internet connection to run it.'
-        )
-      )
```

Comment:
was this error removed on purpose?

Reply:
Yes, errors are now displayed in the `launch` function itself
Not sure how to do one of those inline code snippet thingies, but here's a link:

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/feat/vite/src/backend/legendary/games.ts#L589-L592